packages/maskbook/src/plugins/Trader/SNSAdaptor/trader/SettingsDialog.tsx

```diff
@@ -57,16 +57,16 @@ export function SettingsDialog(props: SettingsDialogProps) {
     const classes = useStylesExtends(useStyles(), props)
 
     const provider = useValueRef(currentTradeProviderSettings)
-    const slippage = useValueRef(currentSlippageTolerance)
+    const slippage = useValueRef(currentSlippageSettings)
     const { pools } = useTradeProviderSettings(provider)
 
     //#region remote controlled dialog
     const { open, closeDialog } = useRemoteControlledDialog(PluginTraderMessages.swapSettingsUpdated)
     //#endregion
 
     const onReset = useCallback(() => {
-        currentTradeProviderSettings.value = TradeProvider.UNISWAP
-        currentSlippageTolerance.value = SLIPPAGE_TOLERANCE_DEFAULT
+        currentTradeProviderSettings.value = TradeProvider.UNISWAP_V2
+        currentSlippageSettings.value = SLIPPAGE_TOLERANCE_DEFAULT
```

Comment:
Rename the constant name as well?
```suggestion
        currentSlippageSettings.value = SLIPPAGE_SETTINGS_DEFAULT
```